proto/public/api.proto

```diff
@@ -125,8 +169,12 @@ message StatTable {
 
 service Api {
   rpc StatSummary(StatSummaryRequest) returns (StatSummaryResponse) {}
-  rpc Version(Empty) returns (VersionInfo) {}
+
   rpc ListPods(Empty) returns (ListPodsResponse) {}
-  rpc SelfCheck(common.healthcheck.SelfCheckRequest) returns (common.healthcheck.SelfCheckResponse) {}
+
   rpc Tap(TapRequest) returns (stream common.TapEvent) {}
+  rpc K8sTap(K8sTapRequest) returns (stream common.TapEvent) {}
```

Comment:
probably better to name this `TapK8s`

Reply:
agree, in general i'm biasing towards not including `k8s` in names.